server/actions/civTurnNotificationAction.js

```diff
@@ -16,54 +16,55 @@ async function civTurnNotificationAction(request, response) {
     return response.send("");
   }
 
+  const gameName = request.body.value1;
   const steamUser = request.body.value2;
   const discordId = await getDiscordId(channel.channelId, steamUser);
 
   const userStr = discordId ? `<@${discordId}>` : steamUser;
-  const message = getRandomTurnNotifcation(userStr);
+  const message = getRandomTurnNotifcation(gameName, userStr);
   postMessage(channel.channelId, message);
   return response.send("");
 }
 
-function getRandomTurnNotifcation(user) {
+function getRandomTurnNotifcation(game, user) {
```

Comment:
oh whoops, just noticed this typo. Can you update Notifcation to Notification here too?